### DISCUSSION TITLE
狗屎阿里云盘，现在不但限制单线程下载速度(大概500k)，还开始限制并发数了

### DISCUSSION THREAD

**Discussion by abandonstudy** [AlistGo/alist#6818] [General]
![image](https://github.com/user-attachments/assets/46ebbb50-d58e-43ba-9836-677fc1f358fb)
单个ip，自用都被限制，烦

**Comment by abandonstudy** [comment]
![image](https://github.com/user-attachments/assets/5bbc6c4c-2979-46ee-81ea-00e68fed8802)
就算是多线程，也只能到不到4MB/S了，而且多线程下2,3个文件，立马被ban

**Reply by abandonstudy** [reply]
![image](https://github.com/user-attachments/assets/6670e305-ef0b-437a-9cd7-11d7304e5a60)
现在甚至只允许单线程了

**Comment by oyal** [comment]
我webdav不能播放好像也是这个原因，昨天个别视频可以播放，今天全都不行了

**Comment by Smoovsky** [comment]
同，不过刚刚恢复了

**Reply by abandonstudy** [reply]
并没恢复，还是这鬼样，看来得换回百度了
![image](https://github.com/user-attachments/assets/839651cb-89ac-4606-8c5b-7eed2aa405a3)
